CMakeLists.txt

```
@@ -36,27 +36,28 @@ else()
 endif()
 
 set(
-    AGE_CORE_INCLUDE_DIRECTORIES
-    ${CMAKE_SOURCE_DIR}/dependencies/cds/include
-    ${CMAKE_SOURCE_DIR}/src/core
+        AGE_CORE_INCLUDE_DIRECTORIES
+        ${CMAKE_SOURCE_DIR}/dependencies/cds/include
+        ${CMAKE_SOURCE_DIR}/src/core
 )
 
 set(
-    CORE_SOURCES
-    ${CMAKE_SOURCE_DIR}/src/core/lang/string/StringRef.cpp
-    ${CMAKE_SOURCE_DIR}/src/core/lang/filesystem/PathAwareFstream.cpp
+        CORE_SOURCES
+        ${CMAKE_SOURCE_DIR}/src/core/lang/string/StringRef.cpp
+        ${CMAKE_SOURCE_DIR}/src/core/lang/filesystem/PathAwareFstream.cpp
+        src/core/intern/QtDefines.hpp
 )
 
 add_library(
-    lib.core
-    STATIC
-    ${CORE_SOURCES}
+        lib.core
+        STATIC
+        ${CORE_SOURCES}
```

Same comment from l:39-l:41
```suggestion
    lib.core
    STATIC
    ${CORE_SOURCES}
```